3.py

```diff
@@ -0,0 +1,8 @@
+# Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+# Пример:
+# - [1.1, 1.2, 3.1, 5, 10.01] => 0.2
+
+list = [1.1, 1.2, 3.1, 5, 10.01]
+for i in range (len (list)):
+    list [i] = list[i] % 1
+print ("%.2f" % (max (list) - min (list)))
```

Comment:
```suggestion
for i in range(len(list)):
    list [i] = list[i] % 1
print("%.2f" % (max(list) - min(list)))
```
Странная закономерность там, где пробелы не нужны, вы их ставите и наоборот.